src/main/java/com/skeleton/common/exception/ErrorCode.java

```diff
@@ -7,6 +7,7 @@
 public enum ErrorCode {
     // 예시)
     // RECRUITMENT_NOT_FOUND(HttpStatus.BAD_REQUEST, "R001", "해당하는 채용공고가 없습니다."),
+    POST_NOT_FOUND(HttpStatus.BAD_REQUEST, "P001", "해당하는 게시물이 없습니다."),
```

Comment:
```suggestion
    POST_NOT_FOUND(HttpStatus.NOT_FOUND, "P001", "해당하는 게시물이 없습니다."),
```

상태코드 잘못된거 발견해서 수정했습니다